pkg/providers/tinkerbell/stack/testdata/expected_with_registry_mirror.yaml

```diff
@@ -0,0 +1,51 @@
+boots:
+  args:
+    - -dhcp-addr=0.0.0.0:67
+    - -osie-path-override=https://anywhere-assests.eks.amazonaws.com/tinkerbell/hook
+  deploy: true
+  env:
+    - name: DATA_MODEL_VERSION
+      value: kubernetes
+    - name: TINKERBELL_TLS
+      value: "false"
+    - name: TINKERBELL_GRPC_AUTHORITY
+      value: 1.2.3.4:42113
+    - name: BOOTS_EXTRA_KERNEL_ARGS
+      value: tink_worker_image=public.ecr.aws/eks-anywhere/tink-worker:latest
```

Comment:
shouldn't this be something like? `tink_worker_image=1.2.3.4:443/eks-anywhere/tink-worker:latest`?